### PR TITLE
test improvements and add oniondev api

### PR DESCRIPTION
**Pull request by KaitCrawford**
While adding the oniondev api I noticed some things about the existing tests that could be improved

### CODE REVIEW

File: mentorship/comms/models.py [hunk 1]
```diff
@@ -91,4 +91,20 @@ def send_message(self, sender_id, recipient_id, content):
             'message': content,
         }
 
-        post_to_api.delay(url, post_data)
+        make_post_request.delay(url, post_data)
+
+
+class OnionDevApi(Api):
+    authId = models.IntegerField()
```
Comment (hodgestar) [inline mentorship/comms/models.py]: Could we add a docstring to this class and a help string to the authId field?

File: mentorship/comms/tests/test_comms_models.py [hunk 2]
```diff
@@ -100,3 +93,34 @@ def receive_ivr_feedback(self):
...
+class ExampleApiTests(CommsApiTestBase):
+    def mk_example_api(self, url='https://example.com'):
+        api = ExampleApi.objects.create(call_url=url, message_url=url)
+        return api
+
+    def test_example_make_call(self):
+        api = self.mk_example_api()
+        with patch('comms.models.make_post_request.delay') as mock_task:
+            api.make_call('0820000000', '0830000000')
+            self.assertTrue(mock_task.called)
```
Comment (hodgestar) [inline mentorship/comms/tests/test_comms_models.py]: We should probably assert that it was called with the correct arguments.

**Comment by KaitCrawford** [comment]
@hodgestar ready for review. I'm trying to do the steps in separate PRs so they are easier to review
cc @TrevorOctober

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**